test/input/beamville/beam.conf

```diff
@@ -117,8 +122,15 @@ beam.events.filterDist = 10000.0
 # Logger Filtering
 ##################################################################
 //Specify log level,If logger empty no logs from MATSIM and other classes.
-// you can specify more than one logger level like INFO,WARN
-beam.levels.loggerLevels = ["DEBUG"]  //valid options INFO,WARN,DEBUG,TRACE,ERROR
+// you can specify more than one logger level like INFO,WARN,DEBUG,TRACE,ERRO
+beam.levels.dependencyLoggingLevels = [""] // for logging by dependencies like MATSim
+
+//Specify one valid option among INFO,WARN,DEBUG,TRACE,ERROR
+beam.levels.beamLoggingLevels = "" // for all logging internal to BEAM+R5
+
+//Specify the package or full classname which you want to control
+beam.packages=["beam","com.conveyal","scala.beam.sim.BeamSim","beam.agentsim.agents.RideHailingManager"]
```

Comment:
So, which loggingLevel param applies to these classes, beamLoggingLevels or dependencyLoggingLevels or both?